tests/acl/test_acl.py

```diff
@@ -574,14 +588,111 @@ def stage(request, duthosts, rand_one_dut_hostname, tbinfo, is_macsec_enabled_fo
         "Egress ACLs are not currently supported on \"{}\" ASICs".format(duthost.facts["asic_type"])
     )
 
-    # Skip ACL egress tests when run on macsec enabled toplogy with braodcom DNX
-    if request.param == "egress" and is_macsec_enabled_for_test and \
-            duthost.facts.get("platform_asic") == "broadcom-dnx":
-        pytest.skip("Egress ACLs not supported with MACSEC on \"{}\" ASICs".format(duthost.facts["asic_type"]))
-
     return request.param
 
 
+@pytest.fixture(scope="class")
+def setup_standby_ports_on_rand_unselected_tor_class_scope(duthosts, rand_selected_dut, rand_unselected_dut,
+                                                           active_active_ports, tbinfo):     # noqa: F811
+    """Config the active-active dualtor that one ToR as active and the other as standby."""
+    if not ('dualtor-aa' in tbinfo['topo']['name'] and active_active_ports):
+        return
+
+    def check_active_active_port_status(duthost, ports, status):
+        logging.debug(f"Check mux status for ports {ports} is {status}")
+        show_mux_status_ret = show_muxcable_status(duthost)
+        logging.debug("show_mux_status_ret: {}".format(
+            json.dumps(show_mux_status_ret, indent=4)))
+        for port in ports:
+            if port not in show_mux_status_ret:
+                return False
+            elif show_mux_status_ret[port]['status'] != status:
+                return False
+        return True
+
+    def _config_the_active_active_dualtor(active_tor, standby_tor, ports, unconditionally=False):
```

Comment:
Can you try reusing dualtor utilities? 
https://github.com/sonic-net/sonic-mgmt/blob/bf17e0ba776d4da6ff3dbbb8967c964f66862370/tests/common/dualtor/dual_tor_utils.py#L301
https://github.com/sonic-net/sonic-mgmt/blob/bf17e0ba776d4da6ff3dbbb8967c964f66862370/tests/common/dualtor/dual_tor_utils.py#L328